site/content/docs/main/vscode.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Velero developer local install and debug with vscode 
+layout: docs
+---
+
+The following example sets up Velero server and client locally for use with the Visual Studio Code.
+
+## Get the latest Velero source code
+```bash
+mkdir VELERO_DEV && pushd VELERO_DEV
+mkdir bin
+git clone https://github.com/vmware-tanzu/velero.git
+pushd velero
+go build ./cmd/velero
+make local
+popd
+cp velero/_output/bin/linux/amd64/velero bin/
+export PATH=$PATH:$PWD/bin
+which velero
+```
+
+**NOTE:** Ensure that the velero client binary that was just built is indeed the one in the $PATH
+
+## Setup a storage plugin
+In this example you will use aws as the storage plugin.
+
+```bash
+# From the VELERO_DEV directory
+mkdir plugins
+mkdir plugin_src && pushd plugin_src
+git clone https://github.com/vmware-tanzu/velero-plugin-for-aws.git
+cd velero-plugin-for-aws/
+make local
+cp _output/bin/linux/amd64/velero-plugin-for-aws ../../plugins/
+popd
+```
+
+## Setup s3 object storage
+
+1. Create a Velero specific credentials file (`credentials-velero`) in your Velero directory:
+
+    ```bash
+    cat <<"EOF" > credentials-velero
+    [default]
+    aws_access_key_id = minio
+    aws_secret_access_key = minio123
+    EOF
+    ```
+
+1. Start the server and the local storage service. In the Velero directory, run:
+
+    ```bash
+    kubectl apply -f velero/examples/minio/00-minio-deployment.yaml
```

Comment:
What is kubectl connected to? local or remote kubernetes?

Reply:
In this case I was using minikube.